packages/styles/scss/components/card-link/index.scss

```diff
@@ -8,21 +8,30 @@
 @import '../../globals/imports';
 @import '../../temp-carbon-expressive/temp-carbon-expressive';
 
-@import 'carbon-components/scss/components/link/link';
 @import 'carbon-components/scss/components/tile/tile';
+@import 'carbon-components/scss/components/link/link';
 
 @mixin card-link {
   .#{$prefix}--card-link {
     color: $text-01;
     text-decoration: none;
     display: flex;
     flex-direction: column;
-    padding: $carbon--spacing-05;
 
     > *:not(:last-child) {
       margin-bottom: $carbon--spacing-07;
     }
 
+    &__wrapper {
+      padding: $spacing-05;
```

Comment:
can you change the token to `$carbon--spacing-05`? That may fix the build error

Reply:
yeah, might be.